src/apis/AsyncStorage/__tests__/index-test.js

```diff
@@ -1,5 +1,66 @@
 /* eslint-env mocha */
+import _ from 'lodash';
```

Comment:
Please import the specific module you need. We're probably fine with a `typeof` check here though